tests/integration_tests/allennlp_tests/example_with_include_package.jsonnet

```diff
@@ -49,8 +49,7 @@ local DROPOUT = std.extVar('DROPOUT');
       bidirectional: true,
     },
   },
-  iterator: {
-    type: 'basic',
+  data_loader: {
```

Comment:
`example_with_environment_variables.jsonnet` also has the `iterator` key. I think we should replace it with `data_loader` too.